xbmc/pvr/channels/PVRChannelGroupInternal.cpp

```diff
@@ -36,7 +36,7 @@ using namespace PVR;
 using namespace EPG;
 
 CPVRChannelGroupInternal::CPVRChannelGroupInternal(bool bRadio) :
-  CPVRChannelGroup(bRadio, bRadio ? PVR_INTERNAL_GROUP_ID_RADIO : PVR_INTERNAL_GROUP_ID_TV, g_localizeStrings.Get(19287)),
+  CPVRChannelGroup(bRadio, -1, g_localizeStrings.Get(19287)),
```

Comment:
As i said, if you use the default ctor of `CPVRChannelGroup` and set the necessary members in the body of this ctor, there is no need to set `m_iGroupId` again, because its default is -1, check the default ctor in `CPVRChannelGroup`.

```c++
CPVRChannelGroupInternal::CPVRChannelGroupInternal(bool bRadio) :
  CPVRChannelGroup(),
  m_iHiddenChannels(0)
{
  m_iGroupType = PVR_GROUP_TYPE_INTERNAL;
  m_bRadio = bRadio;
  m_strGroupName = g_localizeStrings.Get(19287);
}
```